smoke/IotEdgeQuickstart/details/IotedgedLinux.cs

```diff
@@ -65,20 +65,57 @@ static string GetIpAddress()
         }
     }
 
+    public class UriSocks
+    {
+        const int ManagementPort = 15580;
+        const int WorkloadPort = 15581;
+
+        public UriSocks()
+        {
+            this.ConnectManagement = "unix:///var/run/iotedge/mgmt.sock";
```

Comment:
Same as previous comment.  Don't hard-code this value here, you can get the value from config.yaml and override if it is provided explicitly.

Reply:
The default socket values used to be hardcoded in this file, to begin with.
My understanding is that _IoTEdgeQuickStart_ is responsible for creating the _config.yaml_ (and populating the correct socket values). The _config.yaml_ may not exist until _IoTEdgeQuickStart_ is run.

Reply:
IoTEdgeQuickStart is used to install iotedge daemon and will create config.yaml.  After install iotedged, config.yaml will be created with default values.  In configure method https://github.com/Azure/iotedge/blob/6e7321fcb08d0aabf618a756eb58cad57758616c/smoke/IotEdgeQuickstart/details/IotedgedLinux.cs#L188, you can see that it only touches config parameters if they are provided.  Therefore for these URIs, if they are not provided, no action is needed in this method.